ui/frontend/highlighting.ts

```diff
@@ -7,7 +7,7 @@ export function configureRustErrors({
   reExecuteWithBacktrace,
 }) {
   Prism.languages.rust_errors = { // eslint-disable-line camelcase
-    'warning': /^warning:.*$/m,
+    'warning': /^warning(\[E\d+\])?:.*$/m,
```

Comment:
This is a great start!

Do you see a new lines down where it says `'error-explanation': /\[E\d+\]/,`? Can we follow the same pattern here to make the `[E1234]` part a link to the error guide?

Reply:
Cool! Let me try do update it! Thanks!